File: src/main/java/logan/pickpocket/commands/DeveloperCommand.java
package logan.pickpocket.commands;

import logan.api.command.BasicCommand;
import logan.api.command.SenderTarget;
import org.bukkit.entity.Player;

import java.util.Collections;

public class DeveloperCommand extends BasicCommand<Player> {

    public DeveloperCommand() {
        super("developer", "pickpocket.developer", 1, 1,
                new String[]{"dev"}, SenderTarget.PLAYER, "pickpocket",
                Collections.emptyList(),
                "/pickpocket developer giverandom");
    }

    @Override
    public boolean run(Player sender, String[] args, Object data) {
        throw new UnsupportedOperationException("Not yet implemented");
    }
}

File: src/main/java/logan/pickpocket/commands/DeveloperCommand.kt
package logan.pickpocket.commands

import logan.api.command.BasicCommand
import logan.api.command.SenderTarget
import org.bukkit.entity.Player

class DeveloperCommand : BasicCommand<Player>(
    "developer",
    "pickpocket.developer",
    1..1,
    target = SenderTarget.PLAYER,
        usage = """
        /pickpocket developer giverandom
    """.trimIndent(),
    aliases = arrayOf("dev"),
    parentCommand = "pickpocket"
) {
    override fun run(sender: Player, args: Array<out String>, data: Any?): Boolean {
        TODO("Not yet implemented")
    }
}